lib/rules/space-in-parens.js

```diff
@@ -155,24 +150,23 @@ module.exports = {
             }
 
             if (ALWAYS) {
-                return !isCloserException(left);
+                return !isOpenerException(checkPunctuator(right, left));
             }
-            return isCloserException(left);
-
+            return isOpenerException(checkPunctuator(right, left));
         }
 
         /**
-         * Determines if an opener paren should not have an existing space after it
+         * Determines if an opening paren has a space immediately after it
          * @param {Object} left The paren token
          * @param {Object} right The token after it
-         * @returns {boolean} True if the paren should reject the space
+         * @returns {boolean} True if the opening paren has a disallowed space following it
          */
-        function shouldOpenerRejectSpace(left, right) {
-            if (right.type === "Line") {
+        function openerRejectsSpace(left, right) {
```

Comment:
While we're in this space, could we improve the parameter names? Names like `openingParenToken` and `tokenAfterOpeningParen` would be much easier to read than `left` and `right`.